src/deck/drivers/interface/aideck.h

```diff
@@ -0,0 +1,59 @@
+/**
+ *    ||          ____  _ __
+ * +------+      / __ )(_) /_______________ _____  ___
+ * | 0xBC |     / __  / / __/ ___/ ___/ __ `/_  / / _ \
+ * +------+    / /_/ / / /_/ /__/ /  / /_/ / / /_/  __/
+ *  ||  ||    /_____/_/\__/\___/_/   \__,_/ /___/\___/
+ *
+ * Crazyflie control firmware
+ *
+ * Copyright (C) 2021 Bitcraze AB
```

Comment:
Only 4% left of 2021, cutting it close!

Reply:
Not sure what this should say, should I update it to 2022?

Reply:
No I do not think so, I think it should say _when_ the work was done, since Copyright lapses after X years. So it should be updated only on _major_ revisions that alter the work, I think. A bit fuzzy.